README.md

```diff
@@ -65,29 +39,29 @@ REMOVE - *Example(Create a calculator)*
 ## Features
 <Add a list of features>
 - Create a grid UI
-	- Add numbers 0 - 9
-	- Add a plus button
-	- Add a minus button
-	- Add an equals button
-	- Add a multiply button
-	- Add a modulo/remainder operator
-	- Add a decimal button
-	- Add a reset button
-	- Input section to show entered numbers
+    - Add numbers 0 - 9
+    - Add a plus button
+    - Add a minus button
+    - Add an equals button
+    - Add a multiply button
+    - Add a modulo/remainder operator
+    - Add a decimal button
+    - Add a reset button
+    - Input section to show entered numbers
 
 ### Created By: [YOUR NAME](YOUR CHOSEN SOCIAL MEDIA OR WEBSITE) 
 ```
 
 - Download the code
 - Open in your chosen IDE
-- Upload the file to the wwwroot < project-files < beginner/intermediate/advanced depending on the markdown file. Ensure each file for each skill level has the same name.
+- Upload the file to the www root < project-files < beginner/intermediate/advanced depending on the markdown file. Ensure each file for each skill level has the same name.
```

Comment:
This needs to be wwwroot - just to follow the same file hierarchy

Reply:
noted